packages/checkup-plugin-ember-octane/src/hooks/register-tasks.ts

```diff
@@ -1,8 +1,11 @@
 import { Hook } from '@oclif/config';
 import OctaneMigrationStatusTask from '../tasks/octane-migration-status-task';
+import { getPluginName } from '@checkup/core';
 
 const hook: Hook<'register-tasks'> = async function ({ context, tasks }: any) {
-  tasks.registerTask(new OctaneMigrationStatusTask(context));
+  let pluginName = getPluginName(__dirname);
```

Comment:
I don't fully grok what value `getPluginName` provides here, wouldn't the following also work?

```suggestion
  let pluginName = require('../../package').name;
```

Reply:
It would, but I felt like this was more portable, in that we have to use it both in the hooks file, and in tests. We don't know how deep the tests are housed, so it's mildly irritating to have to think about how many directories to hop up.

Reply:
Will resolve this, but feel free to reopen if you think we should discuss more.